contracts/MetadataViews.cdc

```diff
@@ -29,4 +29,86 @@ pub contract MetadataViews {
         pub fun borrowViewResolver(id: UInt64): &{Resolver}
         pub fun getIDs(): [UInt64]
     }
+
+    // Display is a basic view that includes the name and description
+    // of an object. Most objects should implement this view.
+    //
+    pub struct Display {
+        pub let name: String
+        pub let description: String
```

Comment:
Should we include comments about each field do describe what they should/could contain?